evaluators/evaluator_factory.py

```
@@ -82,15 +82,13 @@
                 info[name] = {
                     "name": temp_evaluator.name,
                     "supported_metrics": temp_evaluator.get_supported_metrics(),
-                    "description": cls._get_evaluator_description(name),
-                    "async": True
+                    "description": cls._get_evaluator_description(name)
                 }
             except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)